README.md

```diff
@@ -63,24 +63,24 @@ Build and push the container image to your container registry
 
 NOTE: This command builds and pushes the image directly to a container image repository bypassing local Docker (if it is present). However, it is possible to build the image using [Docker](https://www.docker.com/) with `./gradlew build jibDockerBuild`. Please refer to [Jib plugin documentation](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#build-to-docker-daemon) for further information.
 
-## Deploying to Lightbend Cloudstate
+## Deploying to Akka Serverless
 
-The following steps use `csctl` to deploy the application to [Lightbend Cloudstate](https://docs.lbcs.io/).
+The following steps use `csctl` to deploy the application to [Akka Serverless](https://docs.cloudstate.io/).
```

Comment:
Assume that should be `docs.cloudstate.com`

```suggestion
The following steps use `csctl` to deploy the application to [Akka Serverless](https://docs.cloudstate.com/).
```

Reply:
Thanks for catching, I was obviously in too big of a hurry!